Heine.Mvc.ActionFilters/Extensions/HttpResponseMessageExtensions.cs

```diff
@@ -13,7 +13,7 @@ public static (int statusCode, string reasonPhrase, string version, string conte
                 response.Version?.ToString(),
                 response.Content?.GetType().FullName,
                 HeaderUtilities.GetLoggableHeaders(response.Headers, response.Content?.Headers),
-                response.Content?.ReadAsString(response.Content.Headers)
+                response.Content?.ReadAsString(response.RequestMessage.Headers)
```

Comment:
På response lesing burde ikke headers fra responsen, og ikke requesten, være det som sendes inn for å sensurere?

Reply:
Riktig det, det er bare så mye enklere å bruke request headerene i stede, det er så hassle å sette response headers i controlleren, eller fant ikke en bra måte å gjøre det på. Så intil videre så bruker vi request headers because its easy... :O

Reply:
Men det blir vel feil? Responsmodellen kan jo være en helt annen en requestmodellen?

Reply:
jepp, its not perfect, but good enough so far, tingen er at om de er forskjellige så legge du bare ved properties for begge modellene så er du "sikra" ulempen er om du har en property med samme navn men som du kun ønsker å obfuscate en av veiene f.eks inn men ikke ut, men det er ikke et problem foreløpig... en annen løsning er å lagre json i headeren med hver sin array for properties å obfuscate på vei inn og ut. Eller beste løsning å få lagret det i response headers, men som sagt med ExecuteAsync på ApiController har jeg ikke funnet en god måte å gjøre det på på en slik måte at det blir gjort før LogAllTrafficActionFilter kjøres på vei ut igjen, og dermed så blir alt logga i klartekst...